cs4660/graph/graph.py

```diff
@@ -36,6 +36,29 @@ def construct_graph_from_file(graph, file_path):
     2. for each following line (from second line to last line), add them as edge to graph
     3. return the graph
     """
+    openFile = open(file_path)
+    readFile = openFile.read()
+    splitReadFile = readFile.split("\n")
```

Comment:
Python recommended naming convention is `underscore_case` than `lowerCamelCase` See https://www.python.org/dev/peps/pep-0008/